draft-js-mention-plugin/src/mentionStrategy.js

```diff
@@ -1,12 +1,12 @@
 import { Entity } from 'draft-js';
 
-const findMention = (character) => {
+const findMention = (trigger) => (character) => {
   const entityKey = character.getEntity();
-  return (entityKey !== null && Entity.get(entityKey).getType() === 'mention');
+  return (entityKey !== null && Entity.get(entityKey).getType() === `mention-${trigger}`);
```

Comment:
If we would change the default type everyone who used the `mention` type and saved it to the database would now need to re-write all save data-structures in their database. That's why the default should stick to `mention`.